src/lib/openapi/spec/edge-token-schema.ts

```diff
@@ -6,19 +6,30 @@ export const edgeTokenSchema = {
     type: 'object',
     additionalProperties: false,
     required: ['token', 'projects', 'type'],
+    description:
+        'A representation of a client token, limiting access to [CLIENT](https://docs.getunleash.io/reference/api-tokens-and-client-keys#client-tokens) (used by serverside SDKs) or [FRONTEND](https://docs.getunleash.io/reference/api-tokens-and-client-keys#front-end-tokens) (used by proxy SDKs)',
     properties: {
         projects: {
+            description:
+                'Since a token can have access to multiple projects, but that is represented with a `[]` in the token, this can be used to expand the []',
```

Comment:
A token has to have access a project. It's the tokenString that can have `[]` as its project. This means we have to fetch projects from the database. Technically `['*', 'a']` could be possible, but it would still mean the same as `['*']`. Since 'a' would be subsumed by the wildcard. I don't think I've seen that happen yet